.pre-commit-config.yaml

```diff
@@ -0,0 +1,9 @@
+-   repo: git://github.com/pre-commit/pre-commit-hooks
```

Comment:
Had to include the stuff from the pre-commit hook branch too, because git wouldn't let me commit code without it!